src/Mod/Part/BasicShapes/SkipGui.py

```diff
@@ -0,0 +1,267 @@
+#! python
+# -*- coding: utf-8 -*-
+# SkipGui.py
+# 2022, by Mark Ganson <TheMarkster>
+# LGPL 2.1 or later
+
+# this file is called from c++ in Mod/Part/Gui/Command.cpp
+# from various functions when these commands are activated:
+#   Part::Loft
+#   Part::Sweep
+#   Part::Extrude
+#   Part::Revolve
+# Then if the user has preselected certain objects and subobjects
+# the Gui can be skipped and the document objects directly created
+
+# If False is returned, then the Gui dialog should be shown
+# Else if True is returned, then the dialog can be skipped
+
+import FreeCAD as App
+import FreeCADGui as Gui
+
+def getActStr(document):
+    return f"App.getDocument('{document.Name}').ActiveObject"
```

Comment:
I don't think the object is available when this one is called.  This is because I prefer not to create a variable in the console if it can be avoided.  No, I didn't check if it was available in FreeCAD.  I presumed it wasn't.

Reply:
I misunderstood at first.  Done.